src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -943,12 +943,25 @@ default Set<ExceptionalDeath> deathSafeExceptions() {
         return EnumSet.of(ExceptionalDeath.FIGHT_CAVE, ExceptionalDeath.INFERNO);
     }
 
+    @ConfigItem(
+        keyName = "deathIgnoredRegions",
+        name = "Ignored Region IDs",
+        description = "User-specified, comma-separated Region IDs where deaths should be ignored.<br/>" +
+            "Use the '::DinkRegion' chat command or an online map to obtain the region IDs.<br/>" +
+            "For example, Prifddinas spans 12894, 12895, 13150, 13151",
+        position = 45,
+        section = deathSection
+    )
+    default String deathIgnoredRegions() {
+        return "8781, 12336, 13462, 13463";
```

Comment:
I feel like we can use ones that are just in the middle of nowhere? e.g. in the ocean, most of the time you can't even access those so you should be unable to die in them.

```suggestion
        return "4904, 5930, 6953, 7723";
```

<details>
<summary>locations</summary>

![image](https://github.com/pajlads/DinkPlugin/assets/41973452/9006d3cc-3858-4177-b0cb-6c688a60f5c9)

</details>